tests/integration/courses.test.js

```diff
@@ -213,6 +217,28 @@ describe('COURSES ROUTES - PUT /courses/{_id}', () => {
       });
 
       expect(response.statusCode).toBe(200);
+
+      const course = await Course.findOne({ _id: coursesList[0]._id }).lean();
+
+      expect(course.name).toEqual(payload.name);
+      expect(course.trainer).toEqual(payload.trainer);
+      expect(course.referent).toEqual(payload.referent);
+    });
+
+    it('should return 400 error', async () => {
+      const payload = {
+        name: 'new name',
+        trainer: new ObjectID(),
```

Comment:
Je crois que ce que @cmolle voulait dire dans son comment, c'était que tu peux enlever ces 2 champs du payload car tu veux juste tester la validité du format de téléphone. D'ailleurs, tu pourrais ajouter ds le titre de ton test 'should return 400 error if referent phone number is invalid'